exercises/H-array-literals/exercise.js

```diff
@@ -4,8 +4,8 @@
   Declare some variables assigned to arrays of values
 */
 
-let numbers = []; // add numbers from 1 to 10 into this array
-let mentors; // Create an array with the names of the mentors: Daniel, Irina and Rares
+let numbers = [1,2,3,4,5,6,7,8,9,10]; // add numbers from 1 to 10 into this array
+let mentors = ['Daniel', 'Irina', 'Rares']; // Create an array with the names of the mentors: Daniel, Irina and Rares
```

Comment:
Hi Abdirahim,
I think is work in this way as well, but is good to have space after the comma before the digit.
Great work !